finder-app/Makefile

```diff
@@ -0,0 +1,19 @@
+CC=gcc
+XCC=aarch64-none-linux-gnu-gcc
+MATCHSTR=aarch64-none-linux-gnu-
```

Comment:
@SundarKrishnakumar you should be ale to simplify this logic to use something like `CC=$(CROSS_COMPILE)gcc`